Puppetfile

```diff
@@ -0,0 +1 @@
+mod 'puppetlabs-ruby_task_helper', '0.4.0'
```

Comment:
Do modules usually include a puppetfile with their dependencies?

Reply:
I think probably not.

Reply:
sweet, so we can delete this in favor of cloning the task helper in travis config?